cot/src/response.rs

```diff
@@ -53,22 +58,6 @@ pub trait ResponseExt: Sized + private::Sealed {
     #[must_use]
     fn builder() -> http::response::Builder;
 
-    /// Create a new HTML response.
-    ///
-    /// This creates a new [`Response`] object with a content type of
-    /// `text/html; charset=utf-8` and given status code and body.
-    ///
-    /// # Examples
-    ///
-    /// ```
-    /// use cot::response::{Response, ResponseExt};
-    /// use cot::{Body, StatusCode};
-    ///
-    /// let response = Response::new_html(StatusCode::OK, Body::fixed("Hello world!"));
-    /// ```
-    #[must_use]
-    fn new_html(status: StatusCode, body: Body) -> Self;
```

Comment:
Can we keep this and just mark it as `#[deprecated]`? This is used in pretty much every Cot project and keeping it as-is for now will greatly reduce the friction when upgrading to 0.3 (if not for other people, this will be helpful for us, as it will allow us to gradually migrate to `Html`/`Json`/other types. We can remove this in 0.4.